tutorials/hello-kubernetes/README.md

```diff
@@ -94,9 +94,11 @@ component.dapr.io/statestore created
 
 ## Step 3 - Deploy the Node.js app with the Dapr sidecar
 
+`dapr.io/enable-api-logging: "true"` annotation is added to node.yaml file by default to see the API logs.
```

Comment:
This sentence
"dapr.io/enable-api-logging: "true" annotation is added to node.yaml file by default to see the API logs."

should be after this sentence
"dapr.io/app-id: nodeapp - this assigns a unique id or name to the Dapr application, so it can be sent messages to and communicated with by other Dapr apps."